File: Exercices/Langage C/exo12.c
#include <stdio.h>

main() {
    int na = 0;
    int ht = 0;
    float tva = 0;
    int ttc = 0;

    printf("Entrez le nombre d'article: ");
    scanf("%d", &na);

    printf("Entrez le prix HT: ");
    scanf("%d", &ht);

    printf("Entrez la TVA: ");
    scanf("%f", &tva);

    ttc = na*ht*(1+tva);

    printf("Le prix TTC est de: %d\n", ttc);

    system ("pause");
}

File: Exercices/Java/exo-java/src/exo12.java
import java.util.Scanner;
public class exo12 {

	public static void main(String[] args) {
		
		int na = 0;
	    float ht = 0;
	    float ttc = 0;
	    float tva = 0;
	    
	    Scanner prix = new Scanner(System.in);
	    
        System.out.print("Entrez le nombre d'article: ");
        na = prix.nextInt();
        
        System.out.print("Entrez le prix HT: ");
        ht = prix.nextFloat();
        
        System.out.print("Entrez la TVA: ");
        tva = prix.nextFloat();
        
        prix.close();
        
        ttc = na * ht * (1 + tva);
        
        System.out.printf("Le prix TTC est de %f ", ttc);
 
	}

}
